File: lib-net/src/main/java/com/llj/lib/net/observer/IObserverTag.java
package com.llj.lib.net.observer;

import io.reactivex.disposables.Disposable;

/**
 * ArchitectureDemo
 * describe:
 * author liulj
 * date 2018/5/10
 */
public interface IObserverTag extends ITag {

    Disposable getDisposable();
}

File: lib-net/src/main/java/com/llj/lib/net/observer/IObserverTag.kt
package com.llj.lib.net.observer

import io.reactivex.disposables.Disposable

/**
 * ArchitectureDemo
 * describe:
 * author liulj
 * date 2018/5/10
 */
interface IObserverTag : ITag {

    fun getDisposable() :Disposable
}
